scripts/scil_compute_local_tracking_dev.py

```diff
@@ -211,7 +225,15 @@ def main():
                   .format(len(streamlines), nbr_seeds, str_time))
 
     # save seeds if args.save_seeds is given
-    data_per_streamline = {'seeds': seeds} if args.save_seeds else {}
+    # We seeded (and tracked) in voxmm, corner, but in dipy, they use
+    # vox, center. In other scripts using the seeds (ex,
+    # scil_compute_density_map), this is what they will expect.
+    if args.save_seeds:
+        seeds = [seed / seed_res for seed in seeds]  # to_vox
+        seeds = [seed - 0.5 for seed in seeds]  # to_center
```

Comment:
you can scale and translate inside the same for loop to avoid iterating twice through all seeds.

Reply:
thanks